### DISCUSSION TITLE
Are there any plans for inline databases?

### DISCUSSION THREAD

**Discussion by isle9** [RafaelGB/obsidian-db-folder#143] [General]
**Is your feature request related to a problem? Please describe.**
Currently, when I try to embed the files themselves with the ![[]] syntax, I get the dataview tables shown, but these can't be edited.

**Describe the solution you'd like**
I'd like to be able to embed my databases directly in my files similarly to what Notion allows you to.

**Describe alternatives you've considered**
An alternative is to use the obsidian database plugin, but I don't know how compatible these two are.

**Comment by RafaelGB** [comment]
Hi @isle9 Iam fixing the inline preview of the databases with 1.7.1

Was broken with the Obsidian version update, but was already a feature in https://github.com/RafaelGB/obsidian-db-folder/issues/41

Its read only
https://user-images.githubusercontent.com/11924043/174282739-ef673d0a-ad68-41f2-a534-7c72013460a9.mov

**Comment by RafaelGB** [comment]
it would not be possible today because it requires a different type of view. The table configuration would not be accessible from the file where it was invoked and the editing engine is designed around that view. It would require a new development, but nothing is ruled out in the long term.

**Reply by isle9** [reply]
I guess this could come into play later when you start developing different views then?

The static embed is more than enough for now, for sure.

**Reply by RafaelGB** [reply]
Could be a possibility. The plugin borns as inline database, but it had too much disadvantages and more friction to the user.
With this I wanna say that the logic to do it  its even more simple than the actual plugin, but I want to achieve the `![[]]`option. Feels more natural but could be A LOT of work instead of
```markdown
'''dbfolder
some configuration
'''
```

**Reply by isle9** [reply]
Exactly what I was thinking too. Thank you for the quick answer and for the plugin!

**Reply by crazymedguy** [reply]
Absolutely agree with this. For example, this plugin - https://github.com/tomaszkiewicz/obsidian-database-plugin/ - does it with ```databaseTable``` but I don't really like extra syntax.
